geopandas/tests/test_geom_methods.py

```diff
@@ -777,9 +777,54 @@ def test_make_valid_shapely_pre18(self):
         ):
             s.make_valid()
 
+    @pytest.mark.skipif(
+        compat.SHAPELY_GE_20,
+        reason="concave_hull is implemented for shapely >= 2.0",
+    )
+    def test_concave_hull_not_implemented_shapely_pre2(self):
+        with pytest.raises(
+            NotImplementedError,
+            match=f"shapely >= 2.0 is required, "
+            f"version {shapely.__version__} is installed",
+        ):
+            squares = GeoSeries([self.sq for _ in range(3)])
```

Comment:
You can create a new GeoSeries from squares above so you don't have to create it from list comprehension repeatedly here.